Issues using flatpak-builder. cannot create regular file '/usr/lib/systemd/system/pcscd.service': Read-only file system

I'm trying to compile freerdp but I can't run `flatpak-builder <build dir> packaging/flatpak/com.freerdp.FreeRDP.json` without it spewing errors.

`/usr/bin/install: cannot create regular file '/usr/lib/systemd/system/pcscd.service': Read-only file system`
`/usr/bin/install: cannot create regular file '/usr/lib/systemd/system/pcscd.socket': Read-only file system`
`make[2]: *** [Makefile:360: install-systemdsystemunitDATA] Error 1`
`make[2]: Leaving directory '/run/build/pcsc/etc'`
`make[1]: *** [Makefile:432: install-am] Error 2`
`make[1]: Leaving directory '/run/build/pcsc/etc'`
`make: *** [Makefile:447: install-recursive] Error 1`
`Error: module pcsc: Child process exited with code 2`

I'm following the wiki page on compiling on linux. I was getting the same errors with /usr/lib/systemd/system/udev-control.socket and some other udev stuff in systemd/system/ but I fixed that by downloading and replacing shared-modules with the latest version from github.

--- a/packaging/flatpak/com.freerdp.FreeRDP.json
+++ b/packaging/flatpak/com.freerdp.FreeRDP.json
@@ -58,7 +58,8 @@
             "--disable-libsystemd",
             "--enable-pic",
             "--disable-libusb",
-            "--enable-shared"
+            "--enable-shared",
+            "--with-systemdsystemunitdir=/app/lib/systemd/"
         ],
         "sources": [
             {

--- a/packaging/flatpak/shared-modules/udev/udev-175.json
+++ b/packaging/flatpak/shared-modules/udev/udev-175.json
@@ -6,7 +6,8 @@
     "--disable-logging",
     "--disable-introspection",
     "--disable-keymap",
-    "--disable-mtd_probe"
+    "--disable-mtd_probe",
+    "--with-systemdsystemunitdir=/app/lib/systemd/"
   ],
   "cleanup": [
     "/include",
@@ -14,6 +15,7 @@
     "/libexec",
     "/sbin",
     "/lib/pkgconfig",
+    "/lib/systemd",
     "/man",
     "/share/aclocal",
     "/share/doc",

